### PR TITLE
Adding a triangle obj and function

### PR DESCRIPTION
**Pull request by sorenpeter**

### CODE REVIEW

File: stdlib/shapes.pz [hunk 1]
```diff
@@ -1,3 +1,7 @@
+func triangle(x, y, r)
```
Comment (rumblesan) [inline stdlib/shapes.pz]: not sure what the `r` arg is doing here?

**Comment by rumblesan** [comment]
thanks for this, I'll run it this evening when I'm not at work, though it all looks good. comment is minor and I can't see any reason why I wouldn't merge once I've poked it.

File: stdlib/shapes.pz (hunk 1 above)
Comment (sorenpeter) [reply stdlib/shapes.pz]: i think my idea was to have a rotation parameter.

Comment (rumblesan) [reply stdlib/shapes.pz]: aahh I see, I guess I'd see rotations and translations as being handled outside of the shape functions themselves.

**Comment by rumblesan** [comment]
just tried it out, all working nicely 😄 
thanks for adding this

**Pull request merged**